Tests/Unit/API/V1/ConverterTests.swift

```diff
@@ -357,14 +357,10 @@ class ConverterTests: XCTestCase {
     }
 
     func test_presence() {
-        let samplePresence: Presence = ["a": "str", "b": 10, "c": 0.5, "d": false, "e": ["a", "b", "c"], "f": ""]
+        let samplePresence: PresenceData = ["a": "str", "b": 10, "c": 0.5, "d": false, "e": ["a", "b", "c"], "f": ""]
```

Comment:
Better if we can add a test case for a value type of `AnyValueTypeDictionary` as below.
"q": ["hello": "world"]

Reply:
Type conversion in Converter has been removed from the following PR(https://github.com/yorkie-team/yorkie-ios-sdk/pull/107) The converter will only pass the string.

So We don't need more Test cases.